### PR TITLE
Secure webapps

### PR DESCRIPTION
**Pull request by pmlopes**
This weeks post

@michel-kraemer could you release it?

### CODE REVIEW

File: src/site/blog/posts/vertx3-secure-webapps.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
+---
+title: Writing secure Vert.x Web apps
+date: 2015-09-07
+template: post.html
+author: pmlopes
+draft: false
+---
+
+This is a starting guide for securing vert.x web applications. It is by no means a comprehensive guide on web application security such as OWASP. Standard rules and practices apply to vert.x apps as if they would to any other web framework.
```
Comment (cescoffier) [inline src/site/blog/posts/vertx3-secure-webapps.md]: Could you add a link about OWASP ?

File: src/site/blog/posts/vertx3-secure-webapps.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+    router.route().handler(CookieHandler.create());
+    router.route().handler(SessionHandler
+        .create(LocalSessionStore.create(vertx))
+        .setCookieHttpOnlyFlag(true)
+        .setCookieSecureFlag(true)
+    );
+    router.route().handler(CSRFHandler.create("not a good secret"));
+
+    router.route().handler(ctx -> {
+      ...
```
Comment (cescoffier) [inline src/site/blog/posts/vertx3-secure-webapps.md]: cosmetic, but could you close the block:

```
router.route().handler(ctx -> {
  // ...
});
```

**Comment by cescoffier** [comment]
Really cool article.
Just two small comments.

**Comment by michel-kraemer** [comment]
@pmlopes Sure thing! Let me know when you have revised the article (see Clement's comments)

**Comment by pmlopes** [comment]
sure will do, 1st thing monday!

**Comment by pmlopes** [comment]
@michel-kraemer the article is ready!

**Pull request merged**